ui-test-util/src/main/java/com/example/uitestutil/TestUtils.kt

```diff
@@ -9,15 +9,15 @@ import kotlin.jvm.Throws
 
 @Throws(Throwable::class)
 suspend fun wr(
-    delay: Long = 200,
-    times: Int = 30,
+    delayInMs: Long = 10,
+    times: Int = 2000,
```

Comment:
If I understand right, the `wr` method is applying a "pooling" mechanism. That way, it's more efficient to let the machine _pool_ several times and check for the task's result instead of a few times with a longer delay. (also that seems to fix some of the UI tests flakiness).

Reply:
great thing